### DISCUSSION TITLE
Unable to add users.

### DISCUSSION THREAD

**Discussion by paulmenefee** [community/community#36900] [New to GitHub]
I created an organization and started on the Team plan.  We added a couple of people and decided we didn't need all of the features.  So, I downgraded to the free tier.  Now we are unable to invite users until a year from now.  The message reads "Your downgrade to Free plan will be effective on Sep 28, 2023."  

How can I use the organization now and invite members now?

**Comment by ossanna16** [comment]
Hey there @paulmenefee and thank you for asking a good question! I'm glad you found our community 🙂
